module/VuFind/src/VuFind/I18n/Translator/TranslatorAwareTrait.php

```diff
@@ -101,6 +101,10 @@ public function getTranslatorLocale($default = 'en')
      */
     public function translate($target, $tokens = [], $default = null)
     {
+        if ($this->translator !== null && $this->translator->getLocale() == "keys") {
```

Comment:
You could use `$this->getTranslatorLocale()` here.

Reply:
Done